concent_api/core/models.py

```diff
@@ -684,3 +685,45 @@ class PendingEthereumTransaction(Model):
     s = DecimalField(max_digits=78, decimal_places=0)
 
     created_at = DateTimeField(auto_now_add=True)
+
+
+class DepositAccount(Model):
+    client = ForeignKey(Client)
+    ethereum_public_key = CharField(max_length = ETHEREUM_ADDRESS_LENGTH)
+    created_at = IntegerField()
+
+
+class ConcentUseCase(ChoiceEnum):
```

Comment:
This is added already in `common.constant`, please reuse it. It is `enum.IntEnum` there so it should be enough if you create `IntegerField` for it.